pepper-apis/config/squid.conf

```diff
@@ -0,0 +1,28 @@
+# Access control definitions.
+acl dest_ports port 9243
+acl allowed_domains dstdomain .cloud.es.io
+acl allowed_clients src 69.173.64.0/18
+acl allowed_clients src 10.8.0.0/28
+acl CONNECT method CONNECT
```

Comment:
Remind me: what are these? I think one of them is Broad? Is the other one supposed to be GAE? Comments might be in order here.

Reply:
Added comments.

Reply:
Not seeing the comments. I think one is the Broad, the other the servless connector, right?

Reply:
Please refresh.